plugins/webapi/value/unspentoutputs/handler.go

```diff
@@ -47,9 +46,10 @@ func Handler(c echo.Context) error {
 					ID:       id.String(),
 					Balances: b,
 					InclusionState: utils.InclusionState{
-						Confirmed: true,
-						Conflict:  false,
-						Liked:     true,
+						Solid:     output.Solid(),
+						Confirmed: output.Confirmed(),
+						Conflict:  output.Rejected(),
```

Comment:
I'm not sure if `conflict` is a good choice here. A transaction can be conflicted but liked and confirmed if we voted on it. So maybe just stick to `rejected`?